frontend/src/metabase/admin/people/containers/PeopleListingApp/PeopleListingApp.tsx

```diff
@@ -91,72 +85,65 @@ export function PeopleListingApp({
   }, [hasDeactivatedUsers, updateStatus]);
 
   return (
-    <SettingsPageWrapper title={external ? t`External Users` : t`People`}>
-      <Stack gap={0}>
-        {isAdmin && hasDeactivatedUsers && (
-          <Tabs value={status} onChange={handleTabChange} pl="md">
-            <Tabs.List className={S.tabs}>
-              <Tabs.Tab value={ACTIVE_STATUS.active}>{t`Active`}</Tabs.Tab>
-              <Tabs.Tab
-                value={ACTIVE_STATUS.deactivated}
-              >{t`Deactivated`}</Tabs.Tab>
-            </Tabs.List>
-          </Tabs>
-        )}
-
-        <SettingsSection>
-          <LoadingAndErrorWrapper
-            error={error}
-            loading={isLoading || !currentUser}
-          >
-            <div data-testid="admin-panel">
-              <Flex
-                align="center"
-                gap="xl"
-                justify="space-between"
-                mb="xl"
-                wrap="wrap"
-              >
+    <div>
+      <Group justify="space-between" w="100%" mb="lg">
+        <Title order={1}>{external ? t`External Users` : t`People`}</Title>
+
+        {!external && <PLUGIN_TENANTS.EditUserStrategySettingsButton />}
+      </Group>
+
+      {isAdmin && hasDeactivatedUsers && (
+        <Tabs value={status} onChange={handleTabChange} pl="md">
+          <Tabs.List className={S.tabs}>
+            <Tabs.Tab value={ACTIVE_STATUS.active}>{t`Active`}</Tabs.Tab>
+            <Tabs.Tab
+              value={ACTIVE_STATUS.deactivated}
+            >{t`Deactivated`}</Tabs.Tab>
+          </Tabs.List>
+        </Tabs>
+      )}
+
+      <SettingsSection>
+        <LoadingAndErrorWrapper
+          error={error}
+          loading={isLoading || !currentUser}
+        >
+          <div data-testid="admin-panel">
+            <Group w="100%" justify="space-between" mb="lg" gap="md">
```

Comment:
Applies a consistent layout for people and external users page

<img width="2008" height="854" alt="Image" src="https://github.com/user-attachments/assets/e024a1ac-4c2a-42a5-9f23-0098c729f6b6" />

Reply:
@WiNloSt yes, it applies the same style from tenants listing 👍🏻